### ISSUE TITLE
fix process send payload

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2389]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.28.4](https://github.com/open-wa/wa-automate-nodejs/compare/4.28.3...4.28.4) (2021-12-30)

- 🐛  fix #2389 [`#2389`](https://github.com/open-wa/wa-automate-nodejs/issues/2389)
- Release 4.28.4 [`7f9b421`](https://github.com/open-wa/wa-automate-nodejs/commit/7f9b42108cde7d9c4712db659d034b7da52b875f)
- updated types-only package [`01748e2`](https://github.com/open-wa/wa-automate-nodejs/commit/01748e2157d0f3c8427a0bbc66f3b27ca5e19da6)
